plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/parser/SQLScriptParser.java

```diff
@@ -718,10 +718,22 @@ public static List<SQLQueryParameter> parseParametersAndVariables(SQLParserConte
                             parameters = new ArrayList<>();
                         }
 
+                        String preparedParamName;
+                        String paramMark = paramName.substring(0, 1);
```

Comment:
Isn't the prefix can be of any length?

Reply:
No, it can be only of length 1 for named parameters.